docs/docs/gatsby-config.md

```diff
@@ -85,7 +85,44 @@ See more about [Browser Support](/docs/browser-support/#polyfills) in Gatsby.
 
 ## mapping
 
-TODO
+To query between nodes, Gatsby has a mapping feature which allows you to link two different nodes by id and then you can query with GraphQL. For instance, if you have a couple of blog posts which have author id in the frontmatter:
+
+```
+title: A blog post
+author: Kyle Mathews
+```
+
+And you have a list of authors and their details stored in `authors.yaml`, you can map between `author` in `frontmatter` to id in `authors.yaml` file by:
+
+```
+module.exports = {
+  mapping: {
+    "MarkdownRemark.frontmatter.author": `AuthorYaml`,
```

Comment:
I think this should be `AuthorsYaml` (plural) if the file is named `authors.yaml`